pep-0690.rst

```diff
@@ -468,6 +491,36 @@ and the ``excluding`` argument to ``set_lazy_imports``, if any, to know whether
 a given import will be eager or lazy.
 
 
+Per-module opt-in
+-----------------
+
+Experience with the reference implementation suggests that the most practical
+adoption path for lazy imports is for a specific deployed application to opt-in
+globally, observe whether anything breaks, and opt-out specific modules as
+needed.
+
+It is less practical to achieve robust and significant startup-time or
+memory-use wins by piecemeal application of lazy imports. Generally it would
+require blanket application of the ``__future__`` import to most of the
+codebase, as well as to third-party dependencies (which may be hard or
+impossible.)
+
+However, under some use cases it may be convenient to have a way to enable lazy
+imports whether the application/end user requests it or not. This too can be
+easily achieved::
+
+    from importlib import enable_lazy_imports_in_module
+    enable_lazy_imports_in_module()
+
+After calling ``enable_lazy_imports_in_module()``, every import in the module
```

Comment:
I think we should explicitly reference SPEC 1 and the existing usage in scientific python libraries here, instead of being vague about who might use this.

Reply:
I added a line about SPEC-1 and its use case in scientific python libraries. Not sure if you feel that's enough, but flagging as resolved for now.